datacenter/dtr/2.2/guides/admin/configure/deploy-a-cache.md

```diff
@@ -1,57 +1,168 @@
 ---
-title: Cache Docker images
-description: Learn how to configure DTR to have caching and making pulls faster.
+title: Configure your Docker Content Cache
```

Comment:
I wanted to include the sentence `service that is deployed separately from DTR` because of the confusion reported from field engineers that didn't know that it's a separate service. The importance is not that its a different image, but rather the deployment should be on a different machine because if they are together then the cache is useless.

@pdevine If we do word it as a feature like `Cache Docker images`, then we should change our frontend user settings to reflect that. Because currently it's called `Content Cache` in DTR. What should we do?